src/main/webapp/i18n/en/course.json

```diff
@@ -54,8 +54,15 @@
             "startDate": "Start",
             "endDate": "End",
             "accessGroups": "Access Groups",
-            "onlineCourse": "Online Course",
-            "registrationEnabled": "Student Registration Enabled",
+            "onlineCourse": {
+                "title": "Online Course",
+                "description": "Activate this checkbox for external online courses (e.g. on edX) so that students can use this Artemis course (via the LTI interface). This also allows automatic account creation if enabled by the Artemis system administrator."
+            },
+            "registrationEnabled": {
+                "title": "Student Course Registration Enabled",
+                "description": "Activate this checkbox to allow students to register independently in the course if it is active (between start and end of the course). You can optionally enter a message that students have to confirm during registration."
```

Comment:
Should we add that only users with a mattching can register to the course to avoid confusion?

Reply:
We could do that. However, the problem is that the instructors do not really know about those mechanisms.
Also this regex is really TUM specific. I don't even know if other universities would use this. At the TUM instance, we also have internal users (e.g. from online courses on edx) that should not be able to register and participate in normal university courses. In the TUM scenario, I would specify it like "Activate this checkbox to allow TUM users to register independently..."

So I am not sure how to formulate this in a more generic but still helpful way, that is not confusing, but that would still apply to other universities?

Reply:
Okay, so we would basically omit this declaration as by default, every user can register (as no username pattern is present).
If an administrator adds a pattern, he is responsible to inform other persons that might need this information.